lib/jni/PrivacyGuardHelper.hpp

```diff
@@ -0,0 +1,14 @@
+//
+// Copyright (c) 2015-2020 Microsoft Corporation and Contributors.
+// SPDX-License-Identifier: Apache-2.0
+//
+#include "modules/privacyguard/PrivacyGuard.hpp"
+#include "ctmacros.hpp"
+
+namespace MAT_NS_BEGIN
+{
+    struct PrivacyGuardHelper {
+        static std::shared_ptr<PrivacyGuard> GetPrivacyGuardPtr() noexcept;
```

Comment:
Needed this to allow getting the PG Shared_Ptr in native, without needing to pass around a raw ptr.